tests/blog-posts/add-update.test.js

```diff
@@ -53,4 +71,21 @@ describe('Add update blog posts', function () {
       });
     });
   });
+
+  describe.only('Create and remove multiple blog posts one after another', function () {
+    newPosts.forEach((newPost) => {
+      it(`should add a new blog post then delete it: ${newPost.title}`, async function () {
+        await blogPosts.addPost(newPost).then((response) => {
+          expect(response.status).to.equal(StatusCode.Created);
+          expect(response.data).to.have.property('id');
+        });
+        await blogPosts.getPost(newPost.userId).then((response) => {
+          expect(response.status).to.equal(StatusCode.Ok);
+        });
+        await blogPosts.deletePost(blogPost.id).then((response) => {
+          expect(response.status).to.equal(StatusCode.Ok);
```

Comment:
I don't think you're reassigning blogPost with the blog post you created from `addPost`. You can assign the blog post returned from your `addPost` and then use the blogPost.id from there to delete.

 ```suggestion
        const addedPost = await blogPosts.addPost(newPost).then((response) => {
          expect(response.status).to.equal(StatusCode.Created);
          expect(response.data).to.have.property('id');
          return response.data;
        });
        await blogPosts.getPost(newPost.userId).then((response) => {
          expect(response.status).to.equal(StatusCode.Ok);
        });
        await blogPosts.deletePost(addedPost.id).then((response) => {
          expect(response.status).to.equal(StatusCode.Ok);
        });
```

Reply:
ah, sorry I didn't read it carefully... What's deleted has been changed from `newPost.id` to `newPost.userId` so that it doesn't require to create a new `const addedPost` and delete `addedPost.id`, does it make sense? 
FYI - test results 👇 
![image](https://user-images.githubusercontent.com/28752414/170353103-6cb0549d-7cf1-46d5-8828-f142daef2ea0.png)

Reply:
Yep that's ok. It would be a more realistic example if you would delete the specific post that was created by the post id rather than the user id. There's no certainty that it is that exact post that was deleted in case the user might have more than just that post.